changelog.d/19183_templateable_chronicle_namespace.md

```diff
@@ -0,0 +1,3 @@
+Make the Chronicle namespace a templatable field
```

Comment:
This changelog needs to be named `changelog.d/19183_templateable_chronicle_namespace.enhancement.md`.